src/widgets/settingspages/ExternalToolsPage.cpp

```diff
@@ -106,7 +106,7 @@ ExternalToolsPage::ExternalToolsPage()
         const auto description = new QLabel(
             "You can set custom host for uploading images, like "
             "imgur.com or s-ul.eu.<br>Check " +
-            formatRichNamedLink("https://chatterino.com/help/image-uploader",
+            formatRichNamedLink("https://wiki.chatterino.com/Image%20Uploader/",
```

Comment:
```suggestion
            formatRichNamedLink("https://wiki.chatterino.com/Image%20Uploader",
```
Doing this from web so I don't have to re-open ide, doing it for  https://github.com/Chatterino/wiki/pull/88#issuecomment-867674003

Reply:
so lazy smh